### PR TITLE
Fix custom deps not working on HA 2021.07

### PR DESCRIPTION
**Pull request by marciogranzotto**
2021.07 seems to use python 3.9

### CODE REVIEW

**Review by ludeeus — CHANGES_REQUESTED** [review]
Please update the version <https://github.com/home-assistant/addons-development/blob/master/custom_deps/config.json#L3> and the CHANGELOG.md file to reflect the changes made here.

**Comment by marciogranzotto** [comment]
@ludeeus done!

File: custom_deps/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+# 1.3.0
+- Update Python 3.9
```
Comment (ludeeus) [inline custom_deps/CHANGELOG.md:4]:
```suggestion
- Update Python to 3.9
```

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: custom_deps/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+# 1.3.0
+- Update Python to 3.9
+
```
Comment (ludeeus) [inline custom_deps/CHANGELOG.md:5]:
```suggestion
- Update Python to 3.9
- Set minimum Core version to 2021.7.0
```

File: custom_deps/config.json [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
   "name": "Custom deps deployment",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "slug": "custom_deps",
   "description": "Manage custom Python modules in Home Assistant deps",
   "url": "https://github.com/home-assistant/hassio-addons-development",
   "arch": ["armhf", "armv7", "aarch64", "amd64", "i386"],
   "startup": "once",
   "advanced": true,
+  "homeassistant", "2021.7.0",
```
Comment (ludeeus) [inline custom_deps/config.json:10]:
```suggestion
  "homeassistant": "2021.7.0",
```

**Pull request merged**